src/common/cups_print.c

```diff
@@ -61,6 +61,29 @@ typedef struct dt_prtctl_t
   void *user_data;
 } dt_prtctl_t;
 
+// add helper functions to convert PPD en float numbers to local based numbers for scanf
+// replacechar from: https://stackoverflow.com/questions/32496497/standard-function-to-replace-character-or-substring-in-a-char-array/32496721
+int replacechar (char *str, char orig, char rep)
+{
+  char *ix = str;
+  int n = 0;
+  while ((ix = strchr (ix, orig)) != NULL)
+    {
+      *ix++ = rep;
+      n++;
+    }
+  return n;
+}
+
+void convert_str_to_loc_numbers (char *data)
+{
+  struct lconv *currentLocalConv;
```

Comment:
const struct lconv *currentLocalConv = localeconv();

Reply:
done

Reply:
I have renamed convert_str_to_loc_numbers to dt_convert_str_to_loc_numbers
hope this is Ok.